bundles/specmate-config/config/specmate-config.properties

```diff
@@ -3,10 +3,18 @@
 logging.level = info
 
 # CDO Persistency Settings
+## CDO Common
+cdo.repositoryName = specmate_repository
+
+## CDO Server
+cdo.serverPort = 2036
+
+## CDO Client
+cdo.resourceName = specmate_resource
+cdo.host = localhost:2036
 ## H2
-h2.cdoRepositoryName = specmate_repository
-h2.cdoResourceName = specmate_resource
-h2.cdoJDBCConnection = jdbc:h2:./database/specmate
+h2.jdbcConnection = jdbc:h2:./database/specmate
+
 

```

Comment:
Adapt the oracle config template below to reflect the changes (refactoring of cdo config options). Check also the impact on OracleProviderConfig and OracleProvider (which has currently a compilation error).

Reply:
done